components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/workflow/WorkflowExecutor.java

```diff
@@ -23,12 +23,13 @@
 import org.wso2.carbon.appmgt.impl.dto.WorkflowDTO;
 import org.wso2.carbon.registry.core.utils.UUIDGenerator;
 
+import java.io.Serializable;
 import java.util.List;
 
 /**
  * This is the class that should be extended by each workflow implementation.
  */
-public abstract class WorkflowExecutor {
+public abstract class WorkflowExecutor implements Serializable{
```

Comment:
Does this need to be Serialized? I don't think this class is ever sent over the network.